commit 9739251161fb33aefe7995153e9f1ea218869de2
Author: Sopheap <sopheap15@gmail.com>
Date:   Sun Feb 22 21:03:22 2026 +0700

    fix gene expression plot with incorrect column name from dataset

diff --git a/functions/preprocessing.R b/functions/preprocessing.R
index 353c129..5106115 100644
--- a/functions/preprocessing.R
+++ b/functions/preprocessing.R
@@ -35,12 +35,12 @@ mr_data <- FetchData(
 ) %>%
   clean_names()
 
-# modify life_cycle_stage to include parasite_stage for schizont stage
+# modify life_cycle_stage to include parasite_stages for schizont stage
 mr_data <- mr_data %>%
   mutate(
     life_cycle_stage = case_when(
       life_cycle_stage == "Schizont" ~
-        paste0(life_cycle_stage, " (", parasite_stage, ")"),
+        paste0(life_cycle_stage, " (", parasite_stages, ")"),
       TRUE ~ life_cycle_stage
     )
   )
